Gems/Atom/Feature/Common/Code/Source/DiffuseGlobalIllumination/DiffuseCompositePass.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <DiffuseGlobalIllumination/DiffuseCompositePass.h>
+#include <DiffuseGlobalIllumination/DiffuseProbeGridFeatureProcessor.h>
+
+namespace AZ
+{
+    namespace Render
+    {
+        // --- Dedicated class for disabling ---
+
+        RPI::Ptr<DiffuseCompositePass> DiffuseCompositePass::Create(const RPI::PassDescriptor& descriptor)
+        {
+            RPI::Ptr<DiffuseCompositePass> pass = aznew DiffuseCompositePass(descriptor);
+            return AZStd::move(pass);
+        }
+
+        DiffuseCompositePass::DiffuseCompositePass(const RPI::PassDescriptor& descriptor)
+            : RPI::FullscreenTrianglePass(descriptor)
+        { }
+
+        bool DiffuseCompositePass::IsEnabled() const
+        {
+            const RPI::Scene* scene = GetScene();
+            if (!scene)
+            {
+                return false;
+            }
+			DiffuseProbeGridFeatureProcessor* diffuseProbeGridFeatureProcessor =
```

Comment:
Is there a tab issue here? Alignment seems off

Reply:
I agree with mriegeramzn, please fix the alignment.

Reply:
I see no alignment issue outstanding. Can you make sure you are looking at the latest version of the PR itself and not just the previous comment? That might be the issue.

Reply:
This is how github shows the code to me
<img width="596" alt="image" src="https://user-images.githubusercontent.com/82187279/144888718-06a1ada5-e691-44f2-90d6-dcc9d52005ba.png">

Reply:
In amazon we all have tabs set to 8 spaces in github so that they show up clearly in PR's. This may be the cause of the difference.